app/src/main/res/navigation/navigation_graph.xml

```diff
@@ -664,8 +664,10 @@
         <action
             android:id="@+id/action_order_completed_to_event_detail"
             app:destination="@id/eventDetailsFragment"
-            app:popUpTo="@id/eventsFragment"
-            app:popUpToInclusive="false" />
+            app:popEnterAnim="@anim/slide_in_left"
```

Comment:
What happens if you press back, it should not redirect to order completed fragment again?

Reply:
No it still redirect to back to that one, please check this one
<img src="https://i.ibb.co/Dbx4n2c/ezgif-4-5b4e4155cde4.gif" alt="ezgif-4-5b4e4155cde4" border="0">

Reply:
Cool.

Reply:
Sorry I have not checked Eventbrite completely. :smile: